src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString
+    .split(';')
+    .filter((item) => item.trim() !== '')
+    .reduce((stylesObject, item) => {
+      const pair = item.split(':');
```

Comment:
When splitting the `item` by ':', it would be safer to limit the number of splits to 2 to avoid issues with property values that might contain colons. For example, `const pair = item.split(':', 2);` ensures that only the first colon is used to split the key from the value.